File: Practice/13/С++/primenumber/primenumber.cpp
#include <iostream>
using namespace std;
int main() {
	setlocale(LC_ALL, "Russian");
	int number, condition;
	cout << "Введите число для проверки" << endl;
	cin >> number;
	condition = 0;
	for (int i = 2; i < number; i++) {
		if (number % i == 0) {
			cout << "Составное число" << endl;
			condition = 1;
			break;
		}
	}
	if (condition != 1) {
		cout << "Простое число" << endl;
}
} 

File: Practice/13/Python/primenumber/primenumber.py
print("Введите число для проверки")
number = int(input())
condition = 0
for i in range (2, number): 
	if (number % i == 0): 
		print("Составное число")
		condition = 1
		break
if (condition != 1):
	print("Простое число") 